evm/vm/base.py

```diff
@@ -70,6 +71,10 @@ class BaseVM(Configurable, metaclass=ABCMeta):
     chaindb = None  # type: BaseChainDB
     _state_class = None  # type: Type[BaseState]
 
+    @abstractproperty
```

Comment:
This has been deprecated since `python3.3`

https://docs.python.org/3.5/library/abc.html#abc.abstractproperty

appropriate way to do this is just

```python
@property
@abstractmethod
def the_method(self):
    ...
```

Reply:
Ah! Fixed it. Will merge once CI goes green :fire: